### ISSUE TITLE
Unrecognized option `--auto-packages`

### ISSUE THREAD

**Issue by polentino** [alisiikh/gradle-scalaxb-plugin#12]
After updating plugin to version `0.1.0`, option `autoPackages = true` is not recognized anymore and, in the gradle build log, I got this error:

```
> Task :openadr-model:generateScalaxb
Error: Unknown option --auto-packages
Try --help for more information.
```

**Comment by olisikh** [comment]
There's a typo, instead of `--autopackages`, the `--auto-packages` is used.
The problem place is the `ScalaxbGenTask.groovy` https://github.com/alisiikh/gradle-scalaxb-plugin/blob/master/src/main/groovy/com/github/alisiikh/scalaxb/ScalaxbGenTask.groovy#L124. 

If you want you can provide a PR for this change.

**Comment by olisikh** [comment]
Published 0.1.1, thanks for your PR!

**Comment by polentino** [comment]
Glad to gave helped, thank you too :)
